common_test.go

```diff
@@ -133,14 +136,17 @@ func (s *BaseSuite) GetLocalRepositoryURL(f *fixtures.Fixture) string {
 
 func (s *BaseSuite) TemporalDir() (path string, clean func()) {
 	fs := osfs.New(os.TempDir())
-	path, err := util.TempDir(fs, "", "")
+	relPath, err := util.TempDir(fs, "", "")
 	if err != nil {
 		panic(err)
 	}
 
-	return fs.Join(fs.Root(), path), func() {
```

Comment:
The bug was here, the first return parameter is named `path` so we were assigning the full path to the variable `path` as well as returning it, then when we called `RemoveAll` it was appending the full path to the base base (instead of just the relative path)